views/help.jade

```diff
@@ -5,27 +5,121 @@ block content
-    
-    #one
-    h3 What is CC0 and public domain?
-    p CC0 is a license used by the creator/rights holder of a work to change the copyright state of a work, enabling it to be used freely by anyone worldwide. Public domain is a label of the state of a work that has fallen out of copyright and into the public domain, and can be used freely by the public.
-
-    #two
-    h3 Why does copyright matter?
-    p The majority of worship songs that are sung in churches right now originate from English-speaking countries and are copyrighted. Churches are legally required to pay for the proper licenses to perform, display, print, and even translate and distribute translations of the songs. And license requirements vary depending on the country. We built Glorify.cc because we want to see the Church unleashed from the status quo. We want to see the Church united in worship across languages and cultures. We want to see the Church be able to worship freely. Revelation 7 says that people from every language and every part of the world will be praising God. Glorify.cc enables churches to sing praises in multiple languages together. It's a platform that empowers the global church to share their songs with one another, to contribute translations so others can join in, and to come together and worship freely.
-
-    #three
-    h3 How do I export my songs to PDF or powerpoint?
-    p Create a playlist and add the songs you want to the playlist. When you are ready, go to Playlists, and view the one you wish to export. Click “Export Playlist” and then select the languages you wish to have each song in. Finally, preview the songs and select which format you want to export the songs in - PDF or powerpoint. The file will then be available in your downloads.
-
-    #four
-    h3 How do I contribute a song or translation to Glorify.cc?
-    p To contribute a song or a translation of a song under the CC0 license, go here <link>. You must create an account<link> in order to contribute to Glorify.cc.
+
+    #one.helpPageQuestion
+        h3 What is CC0 and public domain?
+        p.
+            CC0 (or Creative Commons Zero) is “a license used by the creator/rights holder of a work to change the copyright state of a work, enabling it 
+            to be used freely by anyone worldwide.” (Source: <a href=https://creativecommons.org/publicdomain/zero/1.0/>https://creativecommons.org/publicdomain/zero/1.0/</a>)
+        p This license is critical to giving people permission to use creative works freely as in freedom. 
+        p Public domain is a label of the state of a work that has fallen out of copyright and into the public domain, and can be used freely by the public.
+        
+        p Learn more about CC0 on their website: <a href=https://creativecommons.org/publicdomain/zero/1.0/>https://creativecommons.org/publicdomain/zero/1.0</a>
+        p.
+            Learn more about the difference between Public Domain and CC0:
```

Comment:
Convert this to just make "difference between Public Domain and CC0" the link text.